### ISSUE TITLE
Register false - Email exist or weak password!

### ISSUE THREAD

**Issue by SopMar** [hieuapp/android-firebase-chat#11]
Whenever, I am trying to Register an account, everytime, I am getting:
Register false - Email exist or weak password!

Note: I have already registered my app with Firebase console and copied google-services.json within app folder.

So what could be the reasons ? why I am getting this message and is there something I missed.

**Comment by MintuReena** [comment]
enable  Email/Password  from firebase console

**Comment by MintuReena** [comment]
enable Email/password from firebase console

R33N4

**Comment by SopMar** [comment]
Already Enabled ! But still getting same issue while Registering.

Please have a look at this screenshot: https://ibb.co/kYNCNF

**Comment by MintuReena** [comment]
what is project name? in your google service.json file ?

**Comment by SopMar** [comment]
Thanks for prompt reply friend.... :) DONE

STEPS
1. Uninstalled existing app from Device
2. Deleted existing google-services.json from project
3. Again downloaded google-services.json from console and copied in project
4. Run the app and magic happens :) I got what I was trying to get...

Anyways Thank you so much for your support and instant replies.

**Comment by ramli99** [comment]
Kenapa ya??

**Comment by anuppatra** [comment]
I know it's an old thread and am a beginner but @SopMar so. 1) Import project -> 2) connect to firebase -> 3) replace json file makes the app run on my db? no more changes to be done? Thanks in advance.

**Comment by darkIncorporative** [comment]
@anuppatra no.... If u want to connect to your own database 2 3 files still to be changed..... Just remove the json remove the signatures ..... Connect to firebase in studio and then it will automatically redownload the json from your database go to console.firebase.google.com and enable all the required methods including authentication using email...storage and more

**Comment by ramli99** [comment]
Master,,, you have another project??? Please give me for e-learning...

Pada tanggal Rab, 16 Mei 2018 14.07, darkIncorporative <
notifications@github.com> menulis:
